src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -593,13 +594,16 @@ void CC32xxWiFi::wlan_setup_ap(const char *ssid, const char *security_key,
 
 void CC32xxWiFi::connecting_update_blinker()
 {
-    if (!connected)
+    if (ipAcquiredCallback_ != nullptr)
```

Comment:
I don't understand this change. 
- if the caller does not want the blinker to be updated, they should not call connecting_update_blinker
- the logic of this if seems to be inverted?
- this change in logic should probably be represented in a change of the documentation comment

Reply:
fixed